pkgs/os-specific/linux/zfs/default.nix

```diff
@@ -195,6 +199,7 @@ in {
   zfsStable = common {
     # check the release notes for compatible kernels
     kernelCompatible = kernel.kernelAtLeast "3.10" && kernel.kernelOlder "5.12";
+    latestCompatibleLinuxPackages = pkgs.linuxPackages_5_10;
```

Comment:
ZFS 2.0.4 also supports Kernel 5.11
https://github.com/openzfs/zfs/blob/zfs-2.0.4/META#L9

Reply:
Yes, but we removed it because it was EOL